pkg/operator/ceph/csi/spec.go

```diff
@@ -662,3 +681,198 @@ func (r *ReconcileCSI) validateCSIVersion(ownerInfo *k8sutil.OwnerInfo) (*CephCS
 	}
 	return version, nil
 }
+
+func (r *ReconcileCSI) setupCSINetwork() error {
+	ctx := context.TODO()
```

Comment:
The reconciler has a context already `opManagerContext`, you don't need a new one